android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -453,6 +455,7 @@ protected void onResume() {
 
             if (loadedFlag == null || loadedFlag.equals(BY_NAME)) {
                 loadedFlag = BY_NAME;
+                SharedPreferencesUtil.putInt(ConstantStrings.SESSION_MAP_ID, -1);
```

Comment:
It is retrived in the Map Fragment of FDroid build.
https://github.com/fossasia/open-event-android/blob/development/android/app/src/fdroid/java/org/fossasia/openevent/fragments/OSMapFragment.java#L141